src/index.html

```diff
@@ -6,9 +6,67 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home"><img src="images/logo.png"alt="moyo-logo"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+            class="nav__link is-active">
+            Apple
+            </a>
```

Comment:
```suggestion
            <a href="#apple" class="nav__link is-active">
              Apple
            </a>
```

Reply:
If there are 2 or 1 attributes, you can leave them at one line (last time I showed you wrong example, sorry).
If there are 3 or more - every attributes should starts from new line

Nested elements (or text content) should have indentation 2 spaces from its parent indentation

Reply:
Fix it everywhere, it will make your code cleaner